app/routes/delete.js

```diff
@@ -0,0 +1,41 @@
+const joi = require('joi')
+const ViewModel = require('./models/delete')
+const { deleteAgreement } = require('../agreement')
+const cache = require('../cache')
+
+module.exports = [{
+  method: 'GET',
+  path: '/delete',
+  options: {
+    handler: async (request, h) => {
+      return h.view('delete', new ViewModel())
+    }
+  }
+},
+{
+  method: 'POST',
+  path: '/delete',
```

Comment:
Maybe better if the path is

agreement/delete

Reply:
Done